README.md

```diff
@@ -14,10 +15,9 @@ helm repo add cluster-openstack https://eschercloudai.github.io/cluster-openstac
 helm repo update
 ```
 
-Create a secret called `cloud-config` with the necessary credentials for Kubernetes to be able to talk to OpenStack.  You'll need an [OpenStack client config](https://docs.openstack.org/python-openstackclient/latest/configuration/index.html) file to reference which in this example is called `clouds.yaml`:
+In order for CAPI to be able to create clusters, it needs to know which API to connect to along with a set of appropriate credentials.  Craft an [OpenStack client config](https://docs.openstack.org/python-openstackclient/latest/configuration/index.html) long the lines of the following and save it as `clouds.yaml`:
```

Comment:
Minor typo:
... *along the lines of the following ...

Reply:
Well spotted, fixed! 👍